docs/content/page/index.en-us.md

```diff
@@ -10,8 +10,8 @@ draft: false
 
 # What is Gitea?
 
-Gitea is a painless self-hosted Git service. It is similar to GitHub, Bitbucket, and GitLab.
-Gitea is a fork of [Gogs](http://gogs.io). See the [Gitea Announcement](https://blog.gitea.io/2016/12/welcome-to-gitea/)
+Gitea is a painless self-hosted DevOps service. It is similar to GitHub, Bitbucket or GitLab.
```

Comment:
According to :

> GitHub ... is an Internet hosting service for software development and version control using Git.
> Bitbucket Cloud is a Git based code hosting and collaboration ...
> Bitbucket Cloud is a Git-based code and CI/CD tool 
> GitLab is an open source end-to-end software development platform with built-in version control, issue tracking, code review, CI/CD, and more.


I guess the new word "DevOps" is too simple. IMO it's better to mention: Git code hosting, team collaboration and CI/CD.

Reply:
```suggestion
Gitea is a painless self-hosted all-in-one software development service, it includes Git hosting, code review, team collaboration, package registry and CI/CD. It is similar to GitHub, Bitbucket and GitLab.
```